contracts/interfaces/IStrategy.sol

```diff
@@ -8,7 +8,7 @@ interface IStrategy {
     external
     returns (
       uint positionId,
-      uint premiumReceived,
+      uint premium,
       uint collateralAdded
```

Comment:
should we also change this to capitalUsed and capitalReturned?

Reply:
I think `capitalReturned` is kind of misleading, because in case of a short vault, the premium will not be returned to the vault until the end of round

Reply:
but capitalUsed is good. will change this one for sure